apps/emails-and-messages/src/modules/event-handlers/send-event-messages.ts

```diff
@@ -0,0 +1,79 @@
+import { AuthData } from "@saleor/app-sdk/APL";
+import { logger as pinoLogger } from "../../lib/logger";
+import { sendMjml } from "../mjml/send-mjml";
+import { sendSendgrid } from "../sendgrid/send-sendgrid";
+import { appRouter } from "../trpc/trpc-app-router";
+import { MessageEventTypes } from "./message-event-types";
+
+interface SendEventMessagesArgs {
+  recipientEmail: string;
+  channel: string;
+  event: MessageEventTypes;
+  authData: AuthData;
+  payload: any;
+}
+
+export const sendEventMessages = async ({
+  recipientEmail,
+  channel,
+  event,
+  authData,
+  payload,
+}: SendEventMessagesArgs) => {
+  const logger = pinoLogger.child({
+    fn: "sendEventMessages",
+  });
+
+  logger.debug("Function called");
+
+  // get app configuration
+  const caller = appRouter.createCaller({
+    appId: authData.appId,
+    saleorApiUrl: authData.saleorApiUrl,
+    token: authData.token,
+    ssr: true,
+  });
```

Comment:
I'd be carefull with that. I had this problem with invoices. `createCaller` means it will be called with all middleware, including checking for frontend tokens.

Thats why I extracted `get-app-configuration.service.ts` so we reuse services, instead spaghetti-routers

Reply:
As discussed at the daily meeting - I've moved logic to the service which I call directly during the webhook processing. 
In the future, we will improve the tRPC implementation, task #251